### PR TITLE
nodejs-ca: Sync site.json with english version

### PR DESCRIPTION
**Pull request by Tsuna-mi**
This PR update locale/ca/site.json file as its english version.
To review it we need some help from native catalan speakers. Thanks!

(cc @kvhuaxa @franher @axarrafols @jose-lara)
Related with #1778

### CODE REVIEW

File: locale/ca/site.json [hunk 1]
```diff
@@ -71,11 +71,13 @@
     },
     "package-manager": {
       "link": "download/package-manager",
-      "text": "instal·lar Node.js amb gestor de paquets"
+      "text": "instal·lar Node.js mitjançant gestor de paquets"
```
Comment (luisprieto) [inline locale/ca/site.json]: Start with capital I: `Instal·lar...`

**Review by luisprieto — CHANGES_REQUESTED** [review]

**Review by luisprieto — APPROVED** [review]

File: locale/ca/site.json (hunk 1 above)
Comment (franher) [reply locale/ca/site.json]: `mitjançant un gestor de paquets` could be more accurate

**Review by franher — APPROVED** [review]
LGTM :+1:

File: locale/ca/site.json [hunk 2]
```diff
@@ -71,11 +71,13 @@
     },
     "package-manager": {
       "link": "download/package-manager",
-      "text": "instal·lar Node.js amb gestor de paquets"
+      "text": "Instal·lar Node.js mitjançant gestor de paquets"
```
Comment (franher) [inline locale/ca/site.json]: mitjançant **un** gestor de paquets

**Review by franher — COMMENTED** [review]
Thank you for your contribution @Tsuna-mi 

I did a small suggestion, take a look please.

**Comment by Tsuna-mi** [comment]
@fhemberger This also has no conflicts... if you can't merge it, tell me something

**Comment by fhemberger** [comment]
@Tsuna-mi This one looks the same for me. Please go ahead and merge it. Thank you!

**Pull request merged**